src/utils/componentsToHints.ts

```diff
@@ -3,9 +3,10 @@ import omit from 'lodash/omit';
 import parsePropTypes from 'parse-prop-types';
 import type { PlayroomProps } from '../Playroom/Playroom';
 
-const staticTypes = __PLAYROOM_GLOBAL__STATIC_TYPES__;
```

Comment:
Moving this into `Playroom.tsx` makes this function much easier to test as there's no need to modify globals in the tests.